scripts/gcp_caper_server/README.md

```diff
@@ -0,0 +1,80 @@
+## Introduction
+
+`create_instance.sh` will create an instance on Google Cloud Compute Engine in Google your project and configure the instance for Caper with PostgreSQL database and Google Cloud Life Sciences API (`v2beta`).
+
+Google Cloud Life Sciences API is a new API replacing the old deprecating Genomics API (`v2alpha1`). It requires `--gcp-region` to be defined correctly. Check [supported regions](https://cloud.google.com/life-sciences/docs/concepts/locations) for the new API.
+
+## Requirements
+
+Make sure that `gcloud` (Google Cloud SDK CLI) is installed on your system.
+
+Enable the following APIs on your Google Cloud console.
+* Compute Engine API
+* Cloud Storage (DO NOT click on "Create credentials")
+* Cloud Storage JSON API
+* Life Sciences API
+
+Prepare a service account with enough permission to Compute Engine, Cloud Storage, Life Sciences API and Service Account User. Generate a secret key JSON from it and keep it locally on your computer.
```

Comment:
A link to GCP docs here would be useful.

Reply:
Added links and more info to make the doc cleaner.